rmgpy/molecule/molecule.py

```diff
@@ -873,7 +884,7 @@ def get_bond_string(self):
         the atom labels in alphabetical order (i.e. 'C-H' is possible but not 'H-C')
         :return: str
         """
-        bond_symbol_mapping = {0.1: '~', 1: '-', 1.5: ':', 2: '=', 3: '#'}
+        bond_symbol_mapping = {0.05: '~', 0.1: '~', 1: '-', 1.5: ':', 2: '=', 3: '#'}
```

Comment:
What does 0.1 stand for currently? Is it intended for 0.05 reaction bond to share the same symbol as the 0.1 bond?

Reply:
0.1 is an H bond, 0 is vdW bond, now 0.05 is R bond. Yes that's intentional.